integrationTests/integration_test.go

```diff
@@ -85,6 +85,11 @@ func SetUp(b *testing.B) {
 			req.Header.Set("Connection", "Keep-Alive")
 			response, err := client.Do(req)
 
+			if err != nil {
+				b.Error("Error: ", err)
+				b.FailNow()
+			}
```

Comment:
I added error check here. Integration tests fails with panic due to nil pointer dereference.